README.md

```diff
@@ -55,16 +55,16 @@ Function | Description | Example
 `getNodesWithExactRoles` | Returns a list of nodes with only the role(s) specified, ignores nodes that have any additional roles except "*node-role.kubernetes.io/worker*" role. | `{{ (getNodesWithExactRoles "infra").items }}`
 `hasNodesWithExactRoles` | Returns `true` if the cluster contains node(s) with only the role(s) specified, ignores nodes that have any additional roles except "*node-role.kubernetes.io/worker*" role. | `key: {{ (hasNodesWithExactRoles "infra") }}` => `key: true`
 
-## `template-resolver` CLI (Beta)
+## `templateresolver` CLI (Beta)
```

Comment:
@dhaiducek could we please keep the dash? It's easier to read and it follows the `kubectl` subcommand formatting (e.g. `cluster-info`).

Reply:
We could, yes. The caveat there is if you keep the dash, it can't be used as-is as a `kubectl` plugin since plugins are delimited by dashes. It has to be `kubectl-templateresolver`, otherwise it'll treat `template` as a parent command of `resolver`.

But as I'm saying all this, it's embedded in the `policy-cli`, so it'd be a non-issue there.